### DISCUSSION TITLE
Hide Attribute in Custom Resource

### DISCUSSION THREAD

**Discussion by szammyboi** [jakobhellermann/bevy-inspector-egui#125] [Q&A] [answered]
Is there a way to hide a single attribute in a resource that has multiple?

For example, in this code I don't want "board" to show up in the egui but I still want num to.
`pub struct Game {
    pub board: [[u32; 8]; 8]
    #[inspector(min = 0, max = 10)]
    pub num: i32
}`

**Comment by jakobhellermann** [comment, marked as answer]
You can use `#[reflect(ignore)]`. That way the field will be ignored by the inspector (and anything else using `Reflect`).
